ngraph/frontend/paddlepaddle/include/paddlepaddle_frontend/model.hpp

```diff
@@ -19,7 +19,7 @@ namespace ngraph
             class InputModelPDPDImpl;
             std::shared_ptr<InputModelPDPDImpl> _impl;
 
-            std::vector<std::shared_ptr<OpPlacePDPD>> getOpPlaces() const;
+            std::vector<std::shared_ptr<OpPlacePDPD>> returnOpPlaces();
```

Comment:
Do you start to modify class in this method?

Reply:
yes, because if model is cut we need to return smaller set of places